<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1597</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1581</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="Metadata" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SRX Beamline </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>1360</x>
        <y>10</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>nsls-ii.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="submit">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>710</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>5-ID2.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>1201</width>
        <height>611</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="Spectroscopy" name="Xanes">
       <attribute name="title">
        <string>Spectroscopy</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>20</y>
          <width>261</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Estimated Scan Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="scan_time">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget_3">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>70</y>
          <width>563</width>
          <height>511</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_5">
         <widget class="QWidget" name="horizontalLayoutWidget_13">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>440</y>
            <width>491</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QCheckBox" name="plot_layover_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Plot Layover</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="normalized1">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Normalized</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="raw1">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Raw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="count1">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Count</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Canvas" name="results" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>561</width>
            <height>441</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <widget class="QWidget" name="horizontalLayoutWidget_9">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>440</y>
            <width>491</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QCheckBox" name="plot_layover_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Plot Layover</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_40">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="normalized2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Normalized</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="raw2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Raw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="count2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Count</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Canvas" name="results_2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>561</width>
            <height>441</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_7">
         <widget class="QWidget" name="horizontalLayoutWidget_12">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>440</y>
            <width>491</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QCheckBox" name="plot_layover">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Plot Layover</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="normalized3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Normalized</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="raw3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Raw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="count3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Count</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Canvas" name="results_3" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>561</width>
            <height>441</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="Window_" name="XRFMapping">
       <attribute name="title">
        <string>XRF Mapping</string>
       </attribute>
       <widget class="Canvas" name="map_" native="true">
        <property name="geometry">
         <rect>
          <x>735</x>
          <y>10</y>
          <width>451</width>
          <height>559</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>10</y>
          <width>191</width>
          <height>35</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="text">
         <string>  Estimated Scan Time(sec): </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="es_time">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>10</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SRX Beamline </string>
      </property>
     </widget>
     <widget class="QPushButton" name="userdataform">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>90</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Data Form</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1360</x>
        <y>10</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>nsls-ii.png</pixmap>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>720</y>
        <width>1201</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="start_scan">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pause_resume_scan">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pause/Resume Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="abort_scan">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Abort Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stop_scan">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stop Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>120</y>
        <width>361</width>
        <height>591</height>
       </rect>
      </property>
      <widget class="Queue" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>720</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="run_queue">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Run Queue</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_queue">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save Queue</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>90</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Queue Run Time: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="queue_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>5-ID2.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1597</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Metadata</class>
   <extends>QWidget</extends>
   <header>Metadata.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Queue</class>
   <extends>QWidget</extends>
   <header>Queue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spectroscopy</class>
   <extends>QWidget</extends>
   <header>Spectroscopy.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Window_</class>
   <extends>QWidget</extends>
   <header>Window_.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>normalized3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>normalizedgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>672</y>
    </hint>
    <hint type="destinationlabel">
     <x>953</x>
     <y>841</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raw3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rawgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>992</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>996</x>
     <y>879</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>count3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>countgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1129</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>1161</x>
     <y>879</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normalized1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>normalizedgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>845</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>853</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raw1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rawgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>1027</x>
     <y>833</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>count1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>countgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1130</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>1091</x>
     <y>879</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normalized2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>normalizedgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>855</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>879</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>raw2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rawgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>957</x>
     <y>809</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>count2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>countgraph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1122</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>1114</x>
     <y>818</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>tabs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setdefault()</slot>
  <slot>normalizedgraph()</slot>
  <slot>rawgraph()</slot>
  <slot>countgraph()</slot>
  <slot>tabs()</slot>
 </slots>
</ui>
